keyboards/foostan/cornelius/readme.md

```diff
@@ -0,0 +1,19 @@
+# cornelius
+
+![cornelius](https://i.imgur.com/kGoUGZgl.jpg)
+
+*Gasket mounted 40% column staggered keyboard manufactured by Yushakobo*
+
+* Keyboard Maintainer: [foostan](https://github.com/foostan/)
+* Hardware Supported: *Cornelius PCBs*
+* Hardware Availability: *http://github.com/foostan*
+
+Make example for this keyboard (after setting up your build environment):
+
+    make foostan/cornelius:default
+
+Flashing example for this keyboard:
+
+    make foostan/cornelius:default:flash
```

Comment:
Could you add instructions on how to enter the bootloader somewhere here?

Reply:
Do you have an example anywhere?

Reply:
Something like: 
```
The board may be reset into bootloader mode via either the reset button (on the bottom of the PCB) or a configured RESET shortcut (which may vary depending on when the PCB was flashed).
```

Reply:
OK, I see.

Reply:
It's DONE 4428697